broker/import_geo/routes.py

```diff
@@ -62,7 +62,7 @@ def _generate_geo_workbook(geo_or_srp_accession: str):
 
 
 def _import_project_from_workbook(workbook):
-    importer = XlsImporter(IngestApi())
+    importer = XlsImporter(app.ingest_api)
```

Comment:
I think that is an anti-pattern.
The ingest api goes have a set_token / unset token methods.
so we can have just one instance of the class and then use a with block to set_token / unset token for the length of a request.

Additionally the geo route doesn't even need that because the token is taken as a parameter on the next two lines:
``` routes.py
def _import_project_from_workbook(workbook):
    importer = XlsImporter(app.ingest_api)
    token = request.headers.get('Authorization')
    project_uuid, errors = importer.import_project_from_workbook(workbook, token)
```

Reply:
Sorry, didn't mean to intrude but I happen to see this PR comment. This was the PR to give more context on the setting of token https://github.com/ebi-ait/ingest-broker/pull/32 The consequence of using one instance of Ingest API is that it will be unset/set/overriding of another user's token. It should be one instance per request to avoid that. It probably needs some thinking and redesigning of classes.

Reply:
thanks @aaclan-ebi, we had gotten there (one ingest per broker request) before abandoning the feature.